tests/c/var_def_is_use-i1.h

```diff
@@ -7,4 +7,4 @@
 //
 //===----------------------------------------------------------------------===//
 
-extern int extern_global;
+extern int ExternGlobal;
```

Comment:
Do you think global variables should also be in PascalCase? The C++ tests use snake_case, like this:
https://github.com/include-what-you-use/include-what-you-use/blob/182147d52d8c7e610c03dce83b67185fe2ba858b/tests/cxx/extern_variable.cc#L17

Reply:
:facepalm: Thanks! I got a little over-excited.

Reply:
Reverted